core/test/run_models_test.jl

```diff
@@ -742,3 +742,14 @@ end
     @test model isa Ribasim.Model
     @test success(model)
 end
+
+@testitem "model without basin" begin
+    toml_path = normpath(@__DIR__, "../../generated_testmodels/no_basin/ribasim.toml")
+    @test ispath(toml_path)
+    model = Ribasim.run(toml_path)
+    @test model isa Ribasim.Model
+    @test success(model)
+
+    (; p_non_diff) = model.integrator.p
+    @test p_non_diff.basin === nothing
```

Comment:
I guess this model name should start with "invalid_" so it is not expected to run in the run test models CI. You can have a look at the other invalid models to see how we tend to test for error logs etc.